package.json

```diff
@@ -369,6 +371,16 @@
                     "when": "view == azureResourceGroups && viewItem == azureextensionui.azureSubscription",
                     "group": "inline"
                 },
+                {
+                    "command": "azureFunctions.createFunction",
+                    "when": "view == azFuncTree && viewItem == azFuncLocalProject",
```

Comment:
Should the view be `azFuncTree`? I pretty much removed all traces of it-- not azureWorkspace?